go/apps/api/openapi/spec/common/permissionReference.yaml

```
@@ -0,0 +1,29 @@
+    pattern: "^[a-zA-Z0-9_.]+$"
+    description: |
+      Identifies the permission by slug for operations on the permission.
+      Permission slugs use dot notation for hierarchical organization.
+      Slugs must be unique within the workspace and are case-sensitive.
+      More readable than IDs but vulnerable to integration breaks if permissions are renamed.
+      Use IDs for automation and slugs for human-configured integrations.
+    example: documents.write
+additionalProperties: false
+example:
+  id: perm_1234567890abcdef
+  slug: documents.write
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)